packages/blocks/src/_common/components/hover/controller.ts

```diff
@@ -105,7 +105,9 @@ export class HoverController implements ReactiveController {
         return;
       }
 
-      this._abortController?.abort();
```

Comment:
We should not abort it every time since `whenHover` may run its callback multiple times, which may cause the tooltip to flicker.


https://github.com/toeverything/blocksuite/blob/8032acf83e8e7dd22470801f2e18dbd3c434070b/packages/blocks/src/_common/components/hover/when-hover.ts#L10